new-pipeline/dist.html

```diff
@@ -55,7 +55,10 @@ <h2 style="font-size: 18px; margin-bottom: 0">
                 </h2>
                 <hr></hr>
                 <figure class="col-md-12">
-                  <figcaption id="dist-caption"></figcaption>
+                <div class="caption-div">
+                  <figcaption id="dist-caption" class="overflow ellipsis"></figcaption>
```

Comment:
A small request: let's call this `dist-caption-text`, to better tell the two elements apart.

Reply:
Sure!